Event/EventsClassification.gd

```diff
@@ -1,17 +0,0 @@
-extends Node
```

Comment:
You could keep the `EventsClassification` if you want but with the simplified structure. Or you can put everything in `EventBase`